### DISCUSSION TITLE
Permission denied @ dir_s_mkdir - /usr/local/share/icons/hicolor/16x16/apps

### DISCUSSION THREAD

**Discussion by Drjacky** [Homebrew/discussions#3772] [Everyday usage]
`brew install gtk+3`:
> Error: An unexpected error occurred during the `brew link` step
The formula built, but is not symlinked into /usr/local
Permission denied @ dir_s_mkdir - /usr/local/share/icons/hicolor/16x16/apps

Then, again `brew install gtk+3`:
> Warning: gtk+3 3.24.34 is already installed, it's just not linked.
To link this version, run:
  brew link gtk+3
  
Then: `brew link gtk+3`:
> Linking /usr/local/Cellar/gtk+3/3.24.34... Error: Permission denied @ dir_s_mkdir - /usr/local/share/icons/hicolor/16x16/apps

Then: `sudo brew link gtk+3`:
> Error: Running Homebrew as root is extremely dangerous and no longer supported.
As Homebrew does not drop privileges on installation you would be giving all
build scripts full access to your system.

**Comment by SMillerDev** [comment]
You'll need to fix the permissions. Not try and force the override
